mscore/selectdialog.cpp

```diff
@@ -95,6 +97,11 @@ void SelectDialog::setPattern(ElementPattern* p)
             p->staffEnd = -1;
             }
 
+      if (sameDuration->isChecked() && e->isRest()) {
+            r = toRest(e);
```

Comment:
I meant here: const Rest* r = toRest(e)

Reply:
yes, you're right, I don't know why I defined it in the header.